conf/agent.conf.template

```diff
@@ -87,6 +87,8 @@ cygnusagent.sinks.hdfs-sink.file_format = json-column
 cygnusagent.sinks.hdfs-sink.batch_size = 100
 # timeout for batch accumulation
 cygnusagent.sinks.hdfs-sink.batch_timeout = 30
+# number of retries upon persitence error
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/cfdffd1ec5590d6d71889726cdb2510c80b304ce